packages/d3fc-webgl/README.md

```diff
@@ -2,15 +2,403 @@
 
 A collection of WebGL shaders and supporting components used by [d3fc-series](https://github.com/d3fc/d3fc/tree/master/packages/d3fc-series#d3fc-series) to render WebGL series.
 
-This package is still a work in progress and its API has not been finalised. Therefore, it is recommended that this package is not used directly.
+This package is still a work in progress and is not feature complete.
 
-# Shader Naming Convention
+[Main D3FC package](https://github.com/d3fc/d3fc)
+
+## Installing
+
+```bash
+npm install @d3fc/d3fc-webgl
+```
+
+## API Reference
+
+* [Series](#series)
+  * [Common Properties](#common-properties)
+  * [Area](#area)
+  * [Bar](#bar)
+  * [BoxPlot](#boxplot)
+  * [Candlestick](#candlestick)
+  * [ErrorBar](#errorbar)
+  * [Line](#line)
+  * [Ohlc](#ohlc)
+  * [Point](#point)
+* [Attribute Buffer Builders](#attribute-buffer-builders)
+  * [Element Attribute](#element-attribute)
+  * [Adjacent Element Attribute](#adjacent-element-attribute)
+  * [Vertex Attribute](#vertex-attribute)
+  * [Base Attribute](#base-attribute)
+* [Uniform](#uniform)
+* [Buffer Builder](#buffer-builder)
+* [Types](#types)
+* [Element Indices](#element-indices)
+* [Shader Naming Convention](#shader-naming-convention)
+
+This package contains the components needed to render a standard or custom series with WebGL. The standard series share a common API with a typical configuration requiring x and y WebGL scales together with a number of attribute buffers.
+
+### Series
+
+The series components can be used to generate a WebGL series of the relevant chart, all chart types are supported.
+
```

Comment:
I think it's worth a subsection here explaining the need for next/previous attributes for some of the series

Reply:
And another one explaining orient (or the lack of it) and why we use main/cross value and how the values are scaled